pretyping/detyping.ml

```diff
@@ -461,7 +461,10 @@ and detype_r d flags avoid env sigma t =
     | Meta n ->
 	(* Meta in constr are not user-parsable and are mapped to Evar *)
         (* using numbers to be unparsable *)
```

Comment:
I guess you can get rid of this comment along the way, since the numbers are certainly not what make them unparsable.

Reply:
Right. I saw it also then forget to change it. Made an update.